src/server/job_func.c

```diff
@@ -1592,9 +1588,9 @@ resc_resv_alloc(void)
 {
 	resc_resv	*resvp;
 
-	resvp = (resc_resv *)malloc(sizeof(resc_resv));
+	resvp = (resc_resv *) calloc(1, sizeof(resc_resv));
```

Comment:
As we are doing calloc do we still need memset in the below lines.

Reply:
dont know how it became calloc. Reverted back.